### ISSUE TITLE
Clicking a link in a previewed messages destroys the message :(

### ISSUE THREAD

**Issue by alexandrinaw** [recursecenter/community#345]
If there's a link in a message that you're writing, and you want to preview that message, and also make sure the link works, if you simply click the link without command+clicking or middle clicking on the link, it'll take you to that page in the current tab. When you go back, your original message is gone! This was super surprising behavior to me, probably because on github, which has a very similar message preview functionality, the message is preserved when you click on a link and then navigate backwards.

**Comment by zachallaun** [comment]
Yeah, this is a major bummer. Because the message field is controlled by
React and isn't natively controlled by the browser, it loses its state
whenever you navigate away.

The real solution would be to save the fields in localStorage and
repopulate them when you navigate back, but a shorter-term and quicker
solution would be to just open links in new tabs by default. If the first
idea isn't hard, we should just do that, but I know the second idea would
only be a couple lines of code because we're already doing it elsewhere.
